content/docs/fedora-releases-guide.md

```diff
@@ -0,0 +1,138 @@
+---
+title: How to do Fedora releases with Packit
+weight: 9
+---
+# How to do Fedora releases with Packit
+
+Let's split the release process into single steps:
+1. New upstream release 
+2. Uploading archive to lookaside cache
+3. Updating dist-git content 
+4. Koji builds
+5. Bodhi updates
+
+Doing Fedora releases with Packit means utilising our 3 jobs - `propose_downstream`, `koji_build` and 
+`bodhi_update` - where each of the jobs takes care of a different part of the release process.
+
+## Propose downstream job
+For enabling the propose downstream job, you need to have [Packit Service installed](/docs/packit-service) 
+and have a `propose_downstream` job in the configuration file for the given upstream repository.
+The [propose_downstream job](/docs/configuration/#propose_downstream) should be then configured like this:
+
+```yaml
+jobs:
+- job: propose_downstream
+  trigger: release
+  dist_git_branches:
+    - main
+```
+You can adjust the `dist_git_branches` field to include the
+dist-git branches you want to update and also utilise [aliases](/docs/configuration/#aliases) 
+instead of using hardcoded versions.
+
+#### New upstream release
+The process of releasing a new version starts in the upstream repository by creating a 
+new upstream release. Packit gets the information about the newly created release from GitHub,
+loads the config from the release commit and if there is a `propose_downstream` job
+defined, the workflow begins.
+Users with write or admin permissions to the repository can retrigger an
+update via a comment in any open issue:
+
+    /packit propose-downstream
+
+#### Uploading archive to lookaside cache
+The upstream archive needs to be downloaded by Packit at first and then uploaded to the lookaside cache.
+By default, Packit downloads sources defined in the specfile that contain URLs.
+You can override these URLs via [`sources`](/docs/configuration#sources) configuration key.
+
+For Python packages, you can use a 
+[GitHub action](https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows)
+([example setup of Packit itself](https://github.com/packit/packit/blob/main/.github/workflows/pypi-publish.yml))
+that automatically builds and uploads the archive to PyPI 
+on each new release. Then during `propose_downstream`, Packit tries to download the archive from the PyPI and 
+also utilises retries in case the upstream archive is not 
```

Comment:
`utilises retries` may be a bit ambiguous to a regular user. Maybe something like `Packit tries to download the archive from the PyPI. If the download fails, i.e. the upstream archive is not available at the time of running the job, the job is scheduled to be retried later.` (also note the `at the time` rather than `in the time`)